src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Diagnostics/RazorDiagnosticConverterTest.cs

```diff
@@ -122,13 +126,14 @@ public void ConvertSpanToRange_EndsOutsideOfDocument_NormalizesToEnd()
         var expectedRange = new Range
         {
             Start = new Position(0, 6),
-            End = new Position(0, 10)
+            End = new Position(0, 11)
         };
 
         // Act
         var range = RazorDiagnosticConverter.ConvertSpanToRange(sourceSpan, sourceText);
 
         // Assert
+        Assert.Equal("World", sourceText.GetSubTextString(range.AsTextSpan(sourceText)));
```

Comment:
Previously this was returning `Worl`, which I'm pretty sure is just a misunderstanding from the original dev, but also why I would always encourage using strings that humans can understand to validate behaviour, and not magic numbers because it is a well known fact that humans cannot understand numbers, and maths is bad.